doc.go

```diff
@@ -85,6 +85,20 @@ https://godoc.org/google.golang.org/grpc/grpclog for more information.
 For HTTP logging, set the GODEBUG environment variable to "http2debug=1" or "http2debug=2".
 
 
+Inspecting errors
+
+All of the errors returned by the generated clients can be converted into a
```

Comment:
Will this always be "all"? Are we doing REST GAPICs? Also, users probably don't know which clients are generated and which aren't.

Reply:
That is a very good call out. It is true when we start to have to REST GAPICs that this will no longer be true. I will switch the language to "Most". I think the example is still good as I made sure to do the type check to be safe.